source/SH3/system/glprogram.cpp

```diff
@@ -48,6 +48,44 @@ std::string sh3_glprogram::load_error::message() const
     return error;
 }
 
+
+void sh3_glprogram::Load(const std::string& name)
```

Comment:
It is. It's been a while since I've looked over the `glprogram` code, so I kinda forgot haha.